axelrod/plot.py

```diff
@@ -8,7 +9,9 @@
     from mpl_toolkits.axes_grid1 import make_axes_locatable
 except ImportError:
     matplotlib_installed = False
-
+except RuntimeError:
+    matplotlib_installed = False
+    warnings.warn('matplotlib does work with virtual environments on MacOS.')
```

Comment:
Perhaps something like:

"If you are using a virtual environment this might be causing the error: matplotlib does **not**..." 

(Only bold facing the not because I think that's what you mean?)

Just thinking in case a run time error kicks in for a different reason?